internal/vpn/run.go

```diff
@@ -40,6 +41,17 @@ func (l *Loop) Run(ctx context.Context, done chan<- struct{}) {
 			vpnRunner, serverName, err = setupOpenVPN(ctx, l.fw,
 				l.openvpnConf, providerConf, settings, l.ipv6Supported, l.starter, subLogger)
 		} else { // Wireguard
+			// save auth in case of PIA for port forwarding and getting the wg config
+			if *settings.Provider.Name == "private internet access" {
```

Comment:
use the constant `providers.PrivateInternetAccess` instead of writing a string directly